code/implementation/host-stats/src/main/java/io/cattle/platform/host/stats/utils/StatsConstants.java

```diff
@@ -0,0 +1,9 @@
+package io.cattle.platform.host.stats.utils;
+
+public class StatsConstants {
+
+    public static final String LINK_STATS = "stats";
```

Comment:
@cjellick Changing it to camelCase here doesn't change it to camelCase in the links.